flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/ConvFunctionITCase.java

```diff
@@ -0,0 +1,149 @@
+                                        "conv(f5, 36, 16)",
+                                        "116ED2B2FB4",
+                                        STRING().nullable()))
+                        .testResult(
+                                resultSpec(
+                                        call("conv", 1111, 2, 10),
+                                        "conv(1111, 2, 10)",
+                                        "15",
+                                        STRING().nullable()))
+
+                        // 2^64 - 1
+                        .testResult(
+                                resultSpec(
+                                        call("conv", "18446744073709551615", 10, 10),
+                                        "conv('18446744073709551615', 10, 10)",
+                                        "18446744073709551615",
+                                        STRING().nullable()))
+                        .testResult(
+                                resultSpec(
+                                        call("conv", "18446744073709551615", 10, -10),
+                                        "conv('18446744073709551615', 10, -10)",
+                                        "-1",
+                                        STRING().nullable()))
+                        // overflow, return maximum (2^64 - 1), which is consistent with
+                        // Mysql.
+                        .testResult(
+                                resultSpec(
+                                        call("conv", "38446744073709551615", 10, 10),
+                                        "conv('38446744073709551615', 10, 10)",
+                                        "18446744073709551615",
+                                        STRING().nullable()))
+
+                        // args is null
+                        .testResult(
+                                resultSpec(
+                                        call("conv", $("f0"), 10, 2),
+                                        "conv(f0, 10, 2)",
+                                        null,
+                                        STRING().nullable()))
+                        .testResult(
+                                resultSpec(
+                                        call("conv", $("f0"), null, 2),
+                                        "conv(f0, null, 2)",
+                                        null,
+                                        STRING().nullable()))
+
+                        // the input is not a decimal number.
+                        .testResult(
+                                resultSpec(
+                                        call("conv", "This is a test String.", 10, 16),
+                                        "conv('This is a test String.', 10, 16)",
+                                        null,
+                                        STRING().nullable()))
+
+                        // fromBase cannot be negative.
```

Comment:
it is also mentioned that min base is 2 and max is 36
it would be great to see tests for such cases when base is 1 or 37 e.g.